doc/nrf/protocols/matter/getting_started/tools.rst

```diff
@@ -184,6 +204,56 @@ You can also install the ZAP tool using one of the following methods:
 
 Both these methods still require adding the ZAP tool location to the system :envvar:`PATH` environment variables, as detailed in the step 3 above.
 
+.. _ug_matter_gs_tools_cert:
+
+CHIP Certificate Tool
+*********************
+
+Matter's CHIP Certificate Tool, in short chip-cert, is a command-line utility tool for generating and editing Matter certificates such as Certificate Declarations (CD), Device Attestation Certificates (DAC), Product Attestation Intermediate (PAI) certificate, alongside their related keys.
```

Comment:
If you want to list all available certificates which can be generated using chip-cert, then a PAA can also be generated using chip-cert